### ISSUE TITLE
Root element is missing

### ISSUE THREAD

**Issue by mayermart** [PhilippC/keepass2android#1284]
After Keepass2Android was automatically updated to Version 1.08-r1 on my Mobile i can't open a Database synced via FTP.
Syncing seems to work. The Error appears after entering the Password.

On my Tablet running 1.07b-r0 i can open the same Database from the same location.

I tried to reinstall Keepass2Android and reconfigured the FTP Connection, but the error stays.

**Comment by PhilippC** [comment]
is this a self-maintained FTP server? Is it possible to set up a testing account for me (crocoapps@gmail.com)?

**Comment by mayermart** [comment]
I sent you the credentials to a ftp Account on my Server. Its a rented Webspace at a German Hoster.

**Comment by inertal** [comment]
I have the same problem, file is on a FTP server. The file is synchronized (and still does fine) with KeePass 2. 
If you need another test account, drop me a note :-)

**Comment by mayermart** [comment]
Hi @inertal 
@PhilippC seems to have found the problem.
If you switch to the Beta Program in Google Play it should work again.
I do not know when the new version will be released to stable Channel.

**Comment by inertal** [comment]
Tested and confirmed working in the Beta. Thanks @PhilippC  and @mayermart !

**Comment by PhilippC** [comment]
thanks for testing and reporting!

**Comment by ikogan** [comment]
This just started happening to me. I tried to update to the latest beta and it's still going on across two devices. I'm using Nextcloud for synchronization though. I can verify that if I copy the file manually to the local device, it does open. The debug logs from the app show nothing out of the ordinary except for the eventual "Root Element is Missing" stack trace. Nextcloud seems to show reasonable web requests:

```
2021-03-12 15:07:44 | cloud.mydomain.org:80 10.42.1.0 - - [12/Mar/2021:20:07:44 +0000] "GET /remote.php/webdav/Home/Documents/Keepass.kdbx HTTP/1.1" 401 1705 "-" "okhttp/4.10.0-RC1"
-- | --
2021-03-12 15:07:45 | cloud.mydomain.org:80 10.42.1.0 - kogan [12/Mar/2021:20:07:44 +0000] "GET /remote.php/webdav/Home/Documents/Keepass.kdbx HTTP/1.1" 200 64609 "-" "okhttp/4.10.0-RC1"
```

Should I open a separate issue for this one? It just started happening recently and I did notice a new update recently and it looks like I'm on `1.09a-pre1`.
